unblob/handlers/filesystem/squashfs.py

```diff
@@ -216,6 +218,40 @@ class SquashFSv4LEHandler(_SquashFSBase):
     HEADER_STRUCT = "squashfs4_super_block_t"
 
 
+class SquashFSv4BEExtractor(Extractor):
```

Comment:
When we define a new `Extractor`, we should define the `get_dependencies` function if that `Extractor` is depending on a specific command / tool. The `Command` extractor used before returns with the `command` by default.

You can check that `sasquatch-v4be` disappeared from this branch from the output of `unblob --show-external-dependencies`.

You can do something like this:


```suggestion
class SquashFSv4BEExtractor(Extractor):

    EXECUTIBLE = "sasquatch-v4be"

    def get_dependencies(self) -> List[str]:
        return [self.EXECUTIBLE]
     
```

Reply:
Good catch ! Fixed.